tools/stress-cluster/chaos/examples/stress-deployment-example/templates/deploy-job.yaml

```diff
@@ -13,6 +13,6 @@ spec:
             source $ENV_FILE &&
             az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID &&
             az account set -s $AZURE_SUBSCRIPTION_ID &&
-            az group show -g $RESOURCE_GROUP -o json
+            az appconfig show -n $APP_CONFIG_NAME -g $RESOURCE_GROUP -o json
```

Comment:
Are we switching to appconfig show to get more detailed info?

Reply:
I just figured this was slightly more demonstrative, since the example does deploy an appconfig resource. Additionally the appconfig resource consumes the new base name parameter, so it makes sense to orient the example around that I think.